src/main/webapp/list.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<meta charset="utf-8" />
```

Comment:
nit: this file name is too vague. List does not really say much about what the file is. I would rename is lectures.html or lecture-list.html

Reply:
File name present in URL--we want to keep them short